manifests/p/Pcgen/Teamviewer/11/Pcgen.Teamviewer.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Pcgen.Teamviewer

```

Comment:
Please verify that the first part of this file's name is exactly the same as this `PackageIdentifier` - consider copying this and pasting into the file name, to be sure.

Reply:
This package should also be put under `TeamViewer.TeamViewer.Host` as the publisher already exists

Reply:
Hi @stephengillie not sure but on my side it is exactly the same, and its the first time im doing this so no idea what it can be. maybe i should try again from scratch.

Hi @Trenly , since this is a custom installer for our company does it count? basically this is our installer of teamviewer not sure how it works (of course publisher is Teamviewer you are right.